.circleci/config.yml

```diff
@@ -0,0 +1,97 @@
+version: 2.1
+
+executors:
+  ubuntu_1804:
+    docker:
+      - image: circleci/buildpack-deps:bionic
+  macos_1013:
+    macos:
+      xcode: "10.0.0"
+
+commands:
+  setup_ubuntu:
+    description: "Setup Ubuntu environment"
+    parameters:
+      OTP_VERSION:
+        type: string
+        default: "20.3.8.23"
```

Comment:
I'd rather put that as implicit parameter of the 'otp20' job (as `setup_ubuntu` of the otp21)